docs/src/misc.rst

```diff
@@ -227,6 +227,18 @@ API
 
     Changes the current working directory.
 
+.. c:function:: int uv_os_homedir(char* buffer, size_t* size)
+
+    Gets the current user's home directory. On Windows, `uv_os_homedir()`, first
+    checks the `USERPROFILE` environment variable using
+    `GetEnvironmentVariableW()`. If `USERPROFILE` is not set,
+    `SHGetKnownFolderPath()` is called. On all other operating systems,
+    `uv_os_homedir()` first checks the `HOME` environment variable using
+    :man:getenv(3). If `HOME` is not set, :man:`getpwuid_r(3)` is called. The
+    user's home directory is stored in `buffer`. When `uv_os_homedir()` is
+    called, `size` indicates the maximum size of `buffer`. On success, `size` is
+    set to the string length of `buffer`. `uv_os_homedir()` is not thread safe.
```

Comment:
Can you please put the last sentence in a ..warning note block? (http://sphinx-doc.org/markup/para.html)